routers/accounts-router.js

```diff
@@ -0,0 +1,80 @@
+const express = require("express");
+
+const db = require("../data/dbConfig");
+
+const router = express.Router();
+
+router.get("/", (req, res) => {
+  const limit = 10;
+  const sortBy = "id";
+  const sortDir = "asc";
```

Comment:
Allow these to come from the URL parameters, for example:
`localhost:4000/api/accounts?limit=5&sortBy=id&sortDir=desc` and then fish them out from req.query like so:
`const { limit, sortBy, sortDir } = req.query;`